python/astro_metadata_translator/headers.py

```diff
@@ -460,4 +470,22 @@ class or else support automatic translation class determination.
                   instrument, obsid, e)
         translator_modified = False
 
-    return modified or translator_modified
+    was_modified = (corrections_file is not None) or translator_modified
+
+    # Always add a sentinel even if we nothing was updated
```

Comment:
I understand the desire to save a little time by not re-scanning, but adding cards (even ones with a deliberately obscure names) seems confusing.

Reply:
Are you against the entire concept of this ticket or against adding provenance when the check decided that nothing was needed?  Assuming the latter, I was on the fence a bit. On the one hand a new version of metadata fixup would now refuse to fix the header even if it was supposed to (although we could overcome that by checking the value of the MODIFIED header and continue if it was False and the versions differed). On the other hand this does at least record which version of the software decided that no fix up was required.

Reply:
Oh, just adding a key when it wasn't modified.  But I don't feel strongly.